plugins/org.orbisgis.groovyeditor/src/org/orbisgis/ui/editors/groovy/GroovyCompletionProcessor.java

```diff
@@ -46,9 +46,21 @@
 public class GroovyCompletionProcessor implements IContentAssistProcessor {
 	
 	private static final Logger LOGGER = new Logger(GroovyCompletionProcessor.class);
```

Comment:
Updates to this file should go in a dedicated PR as the current one is for the test on `ClassPathHandler`